src/patternfly/components/Table/table-grid.scss

```diff
@@ -152,6 +151,11 @@
     white-space: var(--pf-c-table--cell--WhiteSpace);
   }
 
+  [class*="pf-l"],
+  [class*="pf-l"] > * {
```

Comment:
did you mean to leave this in?

Reply:
I did, but using `minmax()` for grid layout columns _seems_ to fix this. Nice catch!